test/unit/lua_func_adapter.c

```diff
@@ -270,7 +270,7 @@ test_msgpack(void)
 	struct func_adapter *func = func_adapter_lua_create(tarantool_L, idx);
 	struct func_adapter_ctx ctx;
 	func_adapter_begin(func, &ctx);
-	func_adapter_push_msgpack(func, &ctx, mp_buf, mp);
+	func_adapter_push_msgpack(func, &ctx, mp_buf, mp, NULL);
```

Comment:
Let's factor `func_adapter` update out to a separate commit and add a unit test for translation.

Reply:
Done.